salt/utils/event.py

```diff
@@ -586,6 +591,9 @@ def __del__(self):
             self.destroy()
         except Exception as ex:
             log.debug(ex)
+            # as this method can be called when logging is already teared down
+            # just try to stdout it
+            traceback.print_exc(file=sys.stderr)
```

Comment:
To include the formated exception in the log message, pass `exc_info_on_loglevel` to the log call specifying the desired level. We comonly show the traceback on debug log level, so you can also pass `exc_info=True ` to the log call.

Sending the traceback to stderr feels a bit as it was unhandled....